src/main/features/claimant-response/views/confirmation.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% from "externalLink.njk" import externalLink %}
+
+{% set headingVisible = false %}
+{% set heading = 'Confirmation' %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <div class="govuk-box-highlight">
+        <h1 class="bold-large">
+          {% if repaymentPlanOrigin === 'DEFENDANT' %}
```

Comment:
you should be able to use the constant for this
something like:
`MadeBy.Defendant.value`

Reply:
we have third option 'COURT' which is not part of `MadeBy` object being used in front-end. So can not use that and don't want to introduce a new just to be used here.

Reply:
shouldn't it have COURT though as its another option being introduced?

Reply:
type inside `Claim` object for this is `string` and not `MadeBy` (it's only used in form model)